### DISCUSSION TITLE
I need to activate the student benefits on another account.

### DISCUSSION THREAD

**Discussion by valerijoao** [community/community#57048] [GitHub Education] [answered]
### Hi everyone,

Last year, I activated the Student Developer Pack on this account, but I forgot about it and created a new one to post my projects and other work. I would like to know if it's possible to activate the student benefits on this new account.

When I try to activate the benefits on the new account, this message appears:
![image](https://github.com/community/community/assets/102620622/7cec6a99-2dcb-479b-a0fd-945a8ecad20e)

**Comment by danielalyoshin** [comment, marked as answer]
Based on #49875, you might be interested in [merging your two accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts). Hope this helps!
